DiscordBot.DataAccess/Models/EventResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DiscordBot.DataAccess.Models
+{
+    public class EventResponse
+    {
+        public string Emoji { get; }
+        public string ResponseName { get; }
+
+        public EventResponse(string emoji, string responseName)
+        {
+            Emoji = emoji;
+            ResponseName = responseName;
+        }
+
+        protected bool Equals(EventResponse other)
+        {
+            return Emoji == other.Emoji && ResponseName == other.ResponseName;
+        }
+
+        public override bool Equals(object? obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
```

Comment:
Would just `obj == null` work? If so, it's simpler 🙂

Reply:
Yeah I think so. Changed in d04752e 🙂